Userland/Libraries/LibMarkdown/BlockQuote.cpp

```diff
@@ -22,8 +22,11 @@ DeprecatedString BlockQuote::render_to_html(bool) const
 
 Vector<DeprecatedString> BlockQuote::render_lines_for_terminal(size_t view_width) const
 {
-    // FIXME: Indent lines inside the blockquote
-    return m_contents->render_lines_for_terminal(view_width);
+    Vector<DeprecatedString> lines;
+    for (auto& line : m_contents->render_lines_for_terminal(view_width - 4))
```

Comment:
What happens when `view_width` is less than 4?

Reply:
Currently, the console renderer generally ignores the argument, only using it for rendering horizontal rules. But you are right, this would cause an infinite loop when rendering a horizontal rule inside a quote on a very narrow terminal